_author_instructions/01_policies.md

```diff
@@ -38,23 +38,23 @@ Click each type of article for more information for submitting an article of tha
 
 ## Presubmission Letter
 
-Authors should first send a presubmission letter to the [lead editor in the relevant area](http://www.livecomsjournal.org/editorial-board).
-The letter should be no more than one page, and should:
-* Outline the scope of the proposed contribution.
-* Explain how the proposed manuscript is different from existing published work.
-* Note whether the manuscript is adapted from a previous article (and identify the article if so).
-* Explain the expertise that the proposed authors have on the subject.
-* Describe the license that the authors will use that will enable the article to be released freely to the public.
-
-Potential authors submitting a presubmission letter will typically receive an answer within two weeks with either encouragement for a full submission, suggestion to work with previous authors on existing articles, or discouragement.
+Before drafting an article, authors should send a presubmission inquiry letter to the [Lead Editor for the relevant area](http://www.livecomsjournal.org/editorial-board).
+The presubmission inquiry letter should be no more than **one page** in length, and should include:
+* An outline the scope of the proposed contribution
+* An explanation how the proposed manuscript differs from existing published work
+* Whether the manuscript is adapted from a previous article (and identify the article if so)
+* A summary of the expertise that the proposed authors have on the subject
+* The proposed license enabling the article to be released freely to the public (see [Licensing](https://livecomsjournal.github.io/authors/policies/index.html#licensing) below for recommendations)
+
+Potential authors submitting a presubmission letter will typically receive an answer within two weeks with either encouragement for a full submission, a suggestion to engage with previous authors to extend an existing articles, or a suggestion that the proposed article would be more suitable for a different journal.
```

Comment:
different journal or publication route?  It might not be suitable for a journal :)  Or perhaps this is just a gentle way of saying that.

Reply:
Fixed (hopefully)!